source/slides/00_introduction.rst

```diff
@@ -121,6 +121,64 @@ Goals
 - Teach modern Development and Operations practices
 - Survey of important and useful techniques used 
 
+System Administration
+=====================
+
+Systems Administrators
+~~~~~~~~~~~~~~~~~~~~~~
+
+* Responsible for systems (typically servers) running code, applications,
+  and services
+
+  - Keeping applications running (they crash, sometimes a lot)
+  - Updates, Security
+  - Monitoring (performance, uptime, etc)
+  - Logging
+  - Often on-call
+
+* Automates significant amounts of work with infrastructure
+
+  - This enables a small team to administer hundreds or thousands of
+    servers
+
+* involved in infrastructure architecture and decisions
+* can be involved in QA/Development work as well
```

Comment:
This slide is just too big. Can you split it into two? Its still not showing everything.